### PR TITLE
Bugfix/gero 217 allow start range none

### PR DESCRIPTION
**Pull request by dustinbleile**
For the edge cases ERBB2:p.G776_V777insC, ERBB2:p.G776_V777insVGC and ERBB2:p.G776_V777insYVMA
observed in GERO-217

### CODE REVIEW

**Review by creisle — CHANGES_REQUESTED** [review]
Can you add a couple of test cases for these variants to reproduce the problem

File: graphkb/match.py [hunk 1]
```diff
@@ -269,7 +269,7 @@ def positions_overlap(
     if pos is None:
         return True
 
-    start = range_start.get('pos', None)
+    start = range_start.get('pos', None) if range_start else None
```
Comment (dustinbleile) [inline graphkb/match.py:272]: Hi @creisle  - added a test match.match_positional_variant(conn, "ERBB2:p.G776_V777insYVMA")

If this change is inappropriate here, the fix probably belongs around line 320 in compare_positional_variants 'break2Start'

Error occurs when variant:
{'break1Repr': 'p.G776',
 'break1Start': {'@class': 'ProteinPosition', 'pos': 776, 'refAA': 'G'},
 'break2Repr': 'p.V777',
 'break2Start': {'@class': 'ProteinPosition', 'pos': 777, 'refAA': 'V'},
 'reference1': 'ERBB2',
 'type': 'insertion',
 'untemplatedSeq': 'YVMA',
 'untemplatedSeqSize': 4}

is compared with overlap to
{'@class': 'PositionalVariant',
 '@rid': '#157:5403',
 'break1Repr': 'p.G776',
 'break1Start': {'@class': 'ProteinPosition', 'pos': 776, 'refAA': 'G'},
 'break2Repr': 'p.?776',
 'break2Start': None,
 'createdAt': 1621292111312,
 'createdBy': '#30:0',
 'displayName': 'ERBB2:p.G776delGinsVV',
 'germline': False,
 'history': '#159:5352',
 'refSeq': 'G',
 'reference1': '#127:91527',
 'type': '#145:17',
 'untemplatedSeq': 'VV',
 'updatedAt': 1621292262454,
 'updatedBy': '#30:0',
 'uuid': '05ff933f-8d85-4102-9d0b-7b42ff804741'}

Comment (creisle) [reply graphkb/match.py:272]: another thing that's weird here is the second one has a "break2Repr" but no break2.

File: tests/test_match.py [hunk 2]
```diff
@@ -395,7 +395,7 @@ def test_known_indel(self, conn):
         assert 'EGFR mutation' in names
         assert 'EGFR copy variant' not in names
 
-    def test_movel_specific_matches_general(self, conn):
+    def test_novel_specific_matches_general(self, conn):
```
Comment (creisle) [inline tests/test_match.py:398]: It would probably be good idea to have the tests on the `compare_positional_variant` instead with the 2 records you gave in your comment above (strip out the unecessary fields for simplicity)

File: graphkb/match.py (hunk 1 above)
Comment (creisle) [reply graphkb/match.py:272]: ya i think this should be in the other function. Made separate PR https://github.com/bcgsc/pori_graphkb_python/pull/43 since its such a small thing was easier than describing. Can we close this one and use that one instead?

**Comment by dustinbleile** [comment]
> ya i think this should be in the other function. Made separate PR #43 since its such a small thing was easier than describing. Can we close this one and use that one instead?

Agreed - closing this PR
